### PR TITLE
Header

### PR DESCRIPTION
**Pull request by onedebos**

### CODE REVIEW

**Comment by mosaaleb** [comment]
Reviewing.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1376 @@
...
+                                <a href="#">Editor's Notes</a>
+                            </div>
+
+                            <div><a href="#"><img src="assets/featured-4.png" alt="featured"
+                                        class="feature img-fluid"></a>
+                            </div>
+
+                        </div>
+                        <h6 class="mt-2 font-weight-bold featured-stories-mainheadline">
+                            <a href="#" class="feature-hover"> To Our Readers:</a>
```
Comment (mosaaleb) [inline index.html]: Top story section is wrapped with `<section>` tag and its story inside has `<article>` tag for more semantic tags.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1376 @@
...
+                    </h6>
+
+                    <!--top story: Main Image-->
+                    <section>
+                        <article>
+                            <div class="wrap-cover">
+                                <div class="wrap">
+                                    <a href="#">u.s</a>
+                                </div>
+                                <div>
```
Comment (mosaaleb) [inline index.html]: Same idea for more stories section, wrapped around `<section>` and each story has `<article>` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1376 @@
...
+
+            </div>
+        </nav>
+
+        <!--END OF NAV-->
+
+        <!--START OF FEATURED STORIES-->
+        <div class="container-fluid header-section">
+            <div class="row border-top border-muted">
+                <section class="col-xl-4 col-lg-4 col-md-5 col-sm-12 mt-4 order-sm-second order-md-first pl-lg-4">
```
Comment (mosaaleb) [inline index.html]: featured stories section is wrapped around `<section>`, and each story inside it has `<article>` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1376 @@
...
+                                <h6 class="font-weight-bold section-heading">
+                                    <a href="#" class="feature-hover">Coca-Cola and PepsiCo Ditch Ties to Plastics
+                                        Industry
+                                        Association</a>
+                                </h6>
+                                <p class="mt-1 more-stories-p"> The injunction is seen as a victory to opponents of the
+                                    laws,
+                                    who argued that they were not medically necessary and violated Supreme Court
+                                    precedent.
+                                </p>
```
Comment (mosaaleb) [inline index.html]: OPINION section wrapped by `<section>` tags, and each opinion has `<article>` tag around

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1376 @@
...
+                        <div class="p-1 bd-highlight">
+                            <span class="font-weight-bold opinion-main-headline float-right">
+                                <a href="#">The Mueller Report's Biggest Open Secret</a>
+                            </span>
+                            <p class="mt-4 opinion-sub-headline font-weight-bold">BY SETH ABRAMSON</p>
+                        </div>
+                    </article>
+
+                    <article class="d-flex pt-3 bd-highlight border-bottom border-muted">
+                        <a href="#">
```
Comment (mosaaleb) [inline index.html]: LATEST NEWS is wrapped by `<section>`, and each article has `<article>` tag around.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1376 @@
...
+                        </div>
+                    </article>
+
+                    <article class="d-flex bd-highlight border-bottom border-muted pr-1 pb-2 ">
+                        <a href="#">
+                            <img src="assets/latest-news-6.jpg" alt="opinion" class=" pt-2 latest-news-img"></a>
+                        <div class="p-1 bd-highlight">
+                            <span class="font-weight-bold opinion-main-headline float-right pl-2">
+                                <a href="#">History Shows More U.S., U.K. Troops in Iran's 'Backyard' Leads to
+                                    Trouble</a>
```
Comment (mosaaleb) [inline index.html]: IN THE MAGAZINE section is wrapped around `<section>`, and each magazine has `<article>` tag around it.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1376 @@
...
+                    </div>
+
+                </article>
+
+                <article class="col-md-3 pr-0 mr-0">
+                    <div class="magazine-img-wrapper">
+                        <img src="assets/magazine-6.jpg" alt="magazine" class="mg-img-8 mt-3 mb-2  mg-img-5 img-fluid">
+                        <span class="mg-category font-weight-bold section-heading mb-0 pr-0">
+                            Downtime
+                            <span class="mg-section ">
```
Comment (mosaaleb) [inline index.html]: The same semantic tags `<section>` and `<article>` apply to EDITOR PICKS section.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1376 @@
...
+                        years.
+                    </p>
+                </article>
+
+                <article class="col-md-3">
+                    <div class="wrap-cover">
+                        <div class="wrap">
+                            <a href="#">WORLD</a>
+                        </div>
+                        <div>
```
Comment (mosaaleb) [inline index.html]: The same concent of semantic tags `<section>` and `<article>` also applies in FEATURED SLIDESHOWS section.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1376 @@
...
+            </div>
+            <!--END OF SLIDESHOWS SECTION CONTAINER-->
+
+            <section class="container-fluid pl-4 pr-4">
+                <div class="row border-top border-muted mt-4">
+                    <h2 class="section-h2 section-heading font-weight-bold text-danger pt-4">
+                        U.S
+                    </h2>
+
+                </div>
```
Comment (mosaaleb) [inline index.html:1094]: The top part and the botom part of footer need to inside `<footer>` tag

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @onedebos 

### Impressive work :clap:. We still have some minor requirements most of them are related to the use of semantic tags.



### Mobile Screen
#### Nav bar

- [ ] (Mobile screen) has sign in and subscribe button -- as seen on newsweek.
- [ ] The SUBSCRIBE section is kind of messed up in mobile screen and overlapping text with the image. Let's fix it.
subscribe

### Medium Screen

- [ ] The SUBSCRIBE  SECTION is also messed up in the medium screen. Causing the whole page to be very wide.
Screenshot from 2019-07-26 21-30-36


### Large Screen

- [ ] Also the same problem with SUBSCRIBE and NEWSLETTER section
- [ ] Also there is overalapping between logo and newsweek text in the footer.
Screenshot from 2019-07-26 21-36-02

Please submit a new code review once you have finished with these requirements. 


Contact me on slack if you have any questions.
### Happy coding :computer:

**Comment by bolah2009** [comment]
PR currently under review

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,393 @@
...
+.opinion-main-headline a:hover{
+    color:red;
+    text-decoration: none;
+}
+
+/* .container-fluid{
+    padding-left:100px;
+    padding-right:100px;
+} */
+
```
Comment (bolah2009) [inline styles.css:151]: Consider deleting commented out code.

**Review by bolah2009 — APPROVED** [review]
## Nice Job [Adebola](../../) :clap: :1st_place_medal: 

The [clone](https://raw.githack.com/onedebos/newsweek-clone/header/index.html) looks good to me and is responsive. :+1: 


#### Check out the comment below and [these errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fonedebos%2Fnewsweek-clone%2Fheader%2Findex.html) on using an heading (h1-h6) element in an `<article>` and `<section>` elelment. Consider checking them out before merging.

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**